source/braille.py

```diff
@@ -2686,6 +2703,17 @@ def _ackTimeoutResetter(self, param: int):
 			self.display._awaitingAck = False
 			self._writeCellsInBackground()
 
+	def updateShowSelection(self) -> None:

```

Comment:
I think this should go in a separate pr as this is a different issue altogether.

Reply:
It can be regarded as another aspect of issue: braille line is not updated although it should be.

But if separate issue and pull request are needed, no problem.

Reply:
I can't speak for others, but I'd prefer this to be handled separately. I don't think the new function is needed either, you can probably just call the initialDisplay method.